src/Module.zig

```diff
@@ -3880,16 +3869,9 @@ const LowerZon = struct {
                 return self.mod.intern_pool.get(gpa, .{ .aggregate = .{
                     .ty = struct_type,
                     .storage = .{ .elems = values },
-                }});
+                } });
             },
-            .array_init_one,
-            .array_init_one_comma,
-            .array_init_dot_two,
-            .array_init_dot_two_comma,
-            .array_init_dot,
-            .array_init_dot_comma ,
-            .array_init,
-            .array_init_comma => {
+            .array_init_one, .array_init_one_comma, .array_init_dot_two, .array_init_dot_two_comma, .array_init_dot, .array_init_dot_comma, .array_init, .array_init_comma => {
```

Comment:
A trailing `,` after the last item would make `zig fmt` keep your previous formatting.